src/components/TeachingBubble/TeachingBubble.scss

```diff
@@ -0,0 +1,165 @@
+@import '../../common/common';
+
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+//
+// Office UI Fabric
+// --------------------------------------------------
+// Teaching Bubble styles
+
+.ms-TeachingBubble {
+  display: block;
+  max-width: 220px;
+}
+
+.ms-TeachingBubble-header {
+  background: $ms-color-themePrimary;
+  margin-bottom: 8px;
+}
+
+.ms-TeachingBubble-title {
+  margin: 0;
+  @include ms-font-xxl;
+  color: $ms-color-white;
+  padding: 24px 20px;
+  font-weight: $ms-font-weight-semilight;
+}
+
+.ms-TeachingBubble-inner {
+  height: 100%;
+  padding: 0 20px 20px;
+  &:first-child {
+    padding-top: 20px;
+  }
+}
+
+.ms-TeachingBubble-subText {
+  margin: 0;
+  font-size: $ms-font-size-m;
+  color: $ms-color-neutralPrimary;
+  font-weight: $ms-font-weight-semilight;
+  .ms-TeachingBubble--dark & {
+    color: $ms-color-white;
+  }
+}
+
+.ms-TeachingBubble-link {
+  @include ms-font-m;
+  color: $ms-color-neutralPrimary;
+}
+
+.ms-TeachingBubble-actions {
+  position: relative;
+  margin-top: 20px;
+  width: 100%;
+  white-space: nowrap;
+}
+
+
+
+// Carried over from callout
```

Comment:
Is there a reason for retaining these styles here? If not, it'd be best to remove them rather than leaving them commented-out.